### PR TITLE
Extend readme and improve description

### PR DESCRIPTION
**Pull request by JasperCraeghs**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -54,18 +54,66 @@ Installation
...
+The *name* attribute of the element to be itemized is used to build the item ID. Lower case letters get converted to
+upper case, whitespaces get converted to underscores, and *&* characters get converted to *AND*. A valid prefix must
+end with a hyphen to be recognized by the script.
+
+Prefix
+======
+
+Traceability item IDs have a prefix that is unique for the group they belong to, e.g. *ITEST_MY_LIB-*. The``--prefix``
+input argument lets you configure this prefix. It will be prepended to the item names found in the input file to build
+the item ID. By default, the script adds _ITEST-_ or _UTEST-_ for integration or unit test reports, unless the prefixes
```
Comment (Letme) [inline README.rst]: Can you emphasize that it adds underscore? Maybe:

```
the script adds _ITEST-_ or _UTEST-_ (note underscores)
```

File: setup.py [hunk 2]
```diff
@@ -12,7 +12,7 @@
     license='Apache License Version 2.0',
     author='JasperCraeghs',
     author_email='jce@melexis.com',
-    description='Python script for converting xUnit/JUnit XML format to reStructuredText format',
+    description='Python script for converting xUnit/JUnit XML format to RST format with traceable items',
```
Comment (Letme) [inline setup.py]: I liked reStructuredText here... it will be better with hits, so maybe change into: `reStructuredText (.rst)`

**Review by Letme — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (JasperCraeghs) [reply README.rst]: Oops. I meant to use italic type here so I should've added * like so `*ITEST-*`
Fixed in e6b30a472076b51ae38573387da441ab5c069690

File: setup.py (hunk 2 above)
Comment (JasperCraeghs) [reply setup.py]: Changed in b87b6bf9b6a8a8ac86a828c7dd9a45adf31c0052

**Review by Letme — APPROVED** [review]

**Pull request merged**